src/Makefile.am

```diff
@@ -76,7 +76,7 @@ stellar_core_LDADD += $(LIBRUST_STELLAR_CORE) -ldl
 
 $(RUST_CXXBRIDGE):
 	mkdir -p $(RUST_BIN_DIR)
-	$(CARGO) install --root $(RUST_BUILD_DIR) cxxbridge-cmd
+	$(CARGO) install --root $(RUST_BUILD_DIR) cxxbridge-cmd --version 1.0.68
```

Comment:
Fyi if you want to extract this from the `Cargo.lock` file so you don't have to manually keep it in sync, you can use this command:

```
cargo metadata --format-version 1 | jq -r '.packages[] | select(.name == "cxx") | .version'
```

I don't know how anything like that could be applied to the VisualStudio project file, but this would work for the Makefile at least.